tests/tests_bridge_nm.yml

```diff
@@ -0,0 +1,28 @@
+---
+- hosts: all
+  name: Run playbook 'tests_bridge.yml' with nm as provider
+  tasks:
+    - name: Get service facts
+      service_facts: null
+    - name: Set network provider to 'nm'
+      set_fact:
+        network_provider: nm
+    - name: Install NetworkManager
+      package:
+        name: NetworkManager
+        state: present
+    - name: Get NetworkManager version
+      command: rpm -q --qf "%{version}" NetworkManager
+      args:
+        warn: "no" 
+      when: true
+      register: NetworkManager_version
+  vars:
+    network_provider_current: nm
+
+- import_playbook: playbooks/tests_bridge.yml
+  when:
+    - ansible_distribution_major_version != '6'
+    # NetworkManager 1.20.0 introduced ethtool settings support
+    # https://github.com/NetworkManager/NetworkManager/blob/master/NEWS
+    - NetworkManager_version.stdout is version('1.20.0', '>=') 
```

Comment:
This check and the related tasks are not needed, since this test is about bridge support and not ethtools support.

Reply:
I think it's good now.

Reply:
Do I need to remove tests_bridge_other_provider.yml as part of this PR?

Reply:
> Do I need to remove tests_bridge_other_provider.yml as part of this PR?

yes, please

Reply:
Done